skypy/halos/tests/test_colossus.py

```diff
@@ -105,3 +105,53 @@ def test_colossus_mf():
     assert np.all(z_halo <= np.max(z))
     assert np.all(m_halo >= m_min)
     assert np.all(m_halo <= m_max)
+
+
+@pytest.mark.skipif(not HAS_COLOSSUS, reason='test requires colossus')
+@pytest.mark.flaky
+def test_colossus_concentration():
+    from skypy.halos._colossus import concentration
+    from astropy.cosmology import Planck15
+
+    mass = np.logspace(np.log10(1e+14), np.log10(1e+15), 10)
+    model, mdef = 'diemer19', '200c'
+    redshift = 0.5
+    cosmo = Planck15
+    sigma8, ns = 0.82, 0.96
+    c = concentration(mass, mdef, redshift, model, cosmo, sigma8, ns)
+
+    assert len(c) == len(mass)
+
+
+@pytest.mark.skipif(not HAS_COLOSSUS, reason='test requires colossus')
+@pytest.mark.flaky
+def test_colossus_radius():
+    from skypy.halos._colossus import radius
+    from astropy.cosmology import Planck15
+
+    mass, c, redshift = 1e+15, 3, 0.5
+    Delta, mdef = '500c', '200c'
+    cosmo = Planck15
+    sigma8, ns = 0.82, 0.96
+
+    rs, RDelta = radius(mass, c, redshift, mdef, Delta, cosmo, sigma8, ns)
+
+    assert rs > 0
+    assert RDelta > 0
+
+
+@pytest.mark.skipif(not HAS_COLOSSUS, reason='test requires colossus')
+@pytest.mark.flaky
+def test_colossus_Delta_Sigma():
+    from skypy.halos._colossus import Delta_Sigma
+    from astropy.cosmology import Planck15
+
+    mass, c, redshift = 1e+15, 3, 0.5
+    mdef = '200c'
+    cosmo = Planck15
+    sigma8, ns = 0.82, 0.96
+    radius = np.logspace(np.log10(300), np.log10(3000), 10)
+
+    DeltaSigma = Delta_Sigma(mass, c, redshift, mdef, radius, cosmo, sigma8, ns)
+
+    assert len(DeltaSigma) == len(radius)
```

Comment:
In general, is there anything else we can test here? Corner cases? Does any of this follow a formula for certain parameters...?